docs/docs/features/reward-distribution.md

```diff
@@ -16,14 +16,63 @@ This will preserve the `ibc denom` that may already be in use.
 This is especially important for standalone chains transitioning to become consumer chains. 
 For more details, see the [changeover procedure](../consumer-development/changeover-procedure.md).
 
-Once on the provider, the ICS rewards are distributed to the opted in validators and their delegators. 
+Once on the provider, the ICS rewards are distributed to the opted in validators and their delegators.
+Note that rewards are **only** distributed to validators that are opted in and have been validating the consumer chain
+for a continuous number of epochs (see `NumberOfEpochsToStartReceivingRewards` param).
+
 To avoid spam, the provider must whitelist denoms before accepting them as ICS rewards.  
 
+## Reward distribution with power capping
+
+If a consumer chain has set a [validators-power cap](./power-shaping#capping-the-validator-powers), then the total received
```

Comment:
@ve6a, I'm adding more information here on the reward distribution to hopefully make it more understandable to users.